Apps/Sandcastle/gallery/Polar Imagery.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Demonstration of Imagery in polar projections.">
+    <meta name="cesium-sandcastle-labels" content="Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var esri53028wkt = '+proj=cass +lat_0=0 +lon_0=0 +x_0=0 +y_0=0 +a=6371000 +b=6371000 +units=m +no_defs';
+var cassini = new Cesium.Proj4Projection(esri53028wkt);
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    mapProjection : cassini,
+    sceneMode : Cesium.SceneMode.COLUMBUS_VIEW
+});
+var epsg3411wkt = '+proj=stere +lat_0=90 +lat_ts=70 +lon_0=-45 +k=1 +x_0=0 +y_0=0 +a=6378273 +b=6356889.449 +units=m +no_defs';
+var epsg3411bounds = Cesium.Rectangle.fromDegrees(-180.0000, 30.0000, 180.0000, 90.0000);
+var epsg3411mapProjection = new Cesium.Proj4Projection(epsg3411wkt, 1.0, epsg3411bounds);
+
+var tilingScheme = new Cesium.ArbitraryProjectionTilingScheme({
+    mapProjection : epsg3411mapProjection,
+    projectedRectangle : new Cesium.Rectangle(-4194304, -4194304, 4194304, 4194304),
+    numberOfLevelZeroTilesX : 2,
+    numberOfLevelZeroTilesY : 2,
+    imageResolution : new Cesium.Cartesian2(512, 512)
+});
+
+var arcticProvider = new Cesium.WebMapTileServiceImageryProvider({
+    url : 'https://map1.vis.earthdata.nasa.gov/wmts-arctic/wmts.cgi',
+    tileMatrixSetID : 'EPSG3413_250m',
+    layer : 'OSM_Land_Water_Map',
+    tilingScheme : tilingScheme,
+    style : 'default',
+    format : 'image/png',
+    tileWidth : 512,
+    tileHeight : 512
```

Comment:
Add `maximumLevel : 5`. Zooming in on this tileset will flood the console with messages about tiles that failed loading.